### ISSUE TITLE
Sage 200 Search Screen - Unsure of issue source

### ISSUE THREAD

**Issue by iStormUK** [crummysummy/Issues-Visco-Therapy-Legacy#12]
Unsure if this is a result of the data being uploaded going into the wrong fields, or if theres an issue with the filter on the site end.

Sage 200 screen search results show jumbled data underneath it, on the live server anyway, lots of Ns where tracking numbers should be, 'Not dispatched' on items all over.

Also, on several instances, when an issue for one of these orders is filed; the usual details like name, address, order number are not populated on the issue tracking screen, resulting in us needing to add such in the notes field, and manually add these details for any collection/replacement state subsequently made.

I suspect these two issues of being interlinked, I may be wrong, but I have never witnessed this with orders found using the old search method.

**Comment by crummysummy** [comment]
Hey. 

So to confirm, this problem exists in the current live site? 

Can you take some screenshots for my reference please? 

_Sent from my Google Nexus 6P using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by LeeVT** [comment]
Pre-existing Issue

**Comment by iStormUK** [comment]
Highlighted affected column and specific lines. Courier consignment numbers in wrong field, couriers in there in its place. For third parties I could understand, but it seems coinflip like as to which occurs on any daily check.  I have a suspicion this may be due to a minor change in the upload format, maybe a stray/additional comma?..

![capture](https://user-images.githubusercontent.com/16283629/29161660-f4eda670-7dae-11e7-9056-8c02c4e1e1df.PNG)

**Comment by crummysummy** [comment]
This may be related to the issue where the sage user's name is under the unit price.

This was caused by someone messing with data exchange settings.

Lee, please remember to look into this.

@LeeVT 

_Sent from my Google Nexus 6P using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by LeeVT** [comment]
@crummysummy It would appear that DocumentCreatedBy collum appears on some bit not on others regardless of the date - not sure if this is due to it being imported? Everything else is the same compared to now and in April 
--

**Comment by crummysummy** [comment]
@LeeVT Not sure what you mean. 

Please compare the Dataexchange order export from previous of May 19th to Dataexchange settings. Make the required changes within Dataexchange to resolve the issue with the Sage Username being under the UnitPrice.

Once that is done, see if it solves that issue and the issue that has been reported under this ticket with the Order Tracking number appearing under the wrong column.

**Comment by LeeVT** [comment]
@crummysummy I have sent you example spreadsheets and highlighted, some include the Collum document created whilst other doesn't. It is random and has no link to the sage issues on 19th may. Cannot connect to FTP to amend the report but dont want to amend something if this works ok?

**Comment by LeeVT** [comment]
@crummysummy @iStormUK Will need to complete this my end with data exchange. I am not classing this as a priority issue. Will close this issue

**Comment by LeeVT** [comment]
@crummysummy @iStormUK I have fixed the issue in regards to the staff name appearing under unit price, Any further random data issues from tomorrow onwards please report them in separate issue
